src/TLuaInterpreter.cpp

```diff
@@ -5765,9 +5765,9 @@ int TLuaInterpreter::debug( lua_State *L )
     for (int i=0; i<nbargs; i++)
     {
         luaDebugText += (nbargs > 1 ? " (" + QString::number(i+1) + ") " : " ") + lua_tostring( L, i+1 );
-        QColor green = QColor(0,255,0);
-        QColor blue = QColor(0,0,255);
-        QColor black = QColor(0,0,0);
+        QColor green = QColor(Qt::green);
+        QColor blue = QColor(Qt::blue);
+        QColor black = QColor(Qt::black);
```

Comment:
Oh, I see.

I'll go with `auto green = QColor(Qt::green);` as it's pretty much the same thing but more consistent in style.